utils/connect.js

```diff
@@ -0,0 +1,60 @@
+const { Sequelize, DataTypes, Model } = require('sequelize');
+
+const sequelize = new Sequelize({
+    dialect: 'sqlite',
+    storage: './database.sqlite'
+});
+
+const testConnection = async function() {
+    try {
+        await sequelize.authenticate();
+        console.log('Connection has been established successfully.');
+      } catch (error) {
+        console.error('Unable to connect to the database:', error);
+      }
+}
+
+class User extends Model {}
```

Comment:
suggestion: perhaps this could have it's own file in a models folder? `src/models/user.js`